lib/workers/repository/update/branch/index.ts

```diff
@@ -223,7 +224,7 @@ export async function processBranch(
           logger.debug('Branch has been edited but found no PR - skipping');
           return {
             branchExists,
-            prNo: branchPr?.number,
+            prNo: undefined,
```

Comment:
are you sure it's always undefined? 🤔

if yes, then there is a bug 

@rarkins WDYT?

Reply:
I'm sure because it's inside the second part of `if (branchPr) { ... } else { ... }` clause. Type inference mechanism just doesn't let you to access `number` property of `null`.

Reply:
so the property should be removed here. we shouldn't explicit set those to undefined